tests/jax/test_jitting.py

```diff
@@ -0,0 +1,59 @@
+import jax
+import jax_galsim as galsim
+
+# Defining jitting identity
+identity = jax.jit(lambda x: x)
+gsparams = galsim.GSParams(minimum_fft_size=32)
+
+
+def test_gaussian_jitting():
+    # Test Gaussian objects
+    objects = [galsim.Gaussian(half_light_radius=1., flux=0.2,
+                               gsparams=gsparams),
+               galsim.Gaussian(sigma=0.1,  flux=0.2,
+                               gsparams=gsparams),
+               galsim.Gaussian(fwhm=1.0,  flux=0.2,
+                               gsparams=gsparams)]
+
+    # Test equality function from original galsim gaussian.py
+    def test_eq(self, other):
+        return (self.sigma == other.sigma and
+                self.flux == other.flux and
+                self.gsparams == other.gsparams)
+
+    # Check that after jitting the oject is still the same
+    assert all([test_eq(identity(o), o) for o in objects])
+
+
+def test_gaussian_jitting():
+    # Test Gaussian objects
```

Comment:
exponential ?

Reply:
yep.... that's what happens I guess when I copy/paste at 1am...